### ISSUE TITLE
changing  navagation  from headings to tab loses your place

### ISSUE THREAD

**Issue by accessaces** [nvaccess/nvda#8693]
if a user moves through a web page using the   hedder navigation and then wants to tab  through the page focus moves back up the page 
 open a web page with lots of headings

**Comment by DrSooom** [comment]
@accessaces: Please completely fill out the draft, otherwise it is very hard to reproduce your issue. Please also list your used web browsers (incl. version). Thanks.

**Comment by derekriemer** [comment]
[**@accessaces**](https://github.com/accessaces) commented on [Aug 29, 2018, 4:55 PM MDT](https://github.com/nvaccess/nvda/issues/8693#issue-355362741 "2018-08-29T22:55:22Z - Replied by Github Reply Comments"):
> if a user moves through a web page using the   hedder navigation and then wants to tab  through the page focus moves back up the page 
>  open a web page with lots of headings 
> 
> ### Steps to reproduce:
> 
> ### Actual behavior:
> 
> ### Expected behavior:
> 
> ### System configuration:
> 
> #### NVDA Installed/portable/running from source:
> 
> #### NVDA version:
> 
> #### Windows version:
> 
> #### Name and version of other software in use when reproducing the issue:
> 
> #### Other information about your system:
> 
> ### Other questions:
> 
> #### Does the issue still occur after restarting your PC?
> 
> #### Have you tried any other versions of NVDA?

Lucy didn't realize the template was below her cursor when editing this issue. I'm going to reword this to ensure a developer can fix it.We're closing this because I can't actually reproduce it and the issue template needs to be completely filled out.
